README.md

```diff
@@ -7,6 +7,37 @@
 
 A PacMan clone written in C++ using SFML, with local multiplayer support.
 
+## Requirements for Building
+
+- C++17 (or newer) toolset.
+- Pre-built [SFML 2.5.x](https://www.sfml-dev.org/download/sfml/2.6.1/) or newer binaries. (Must match your Compiler and OS)
```

Comment:
I think this should say "SFML `2.6.x`"

Reply:
Yes i will fix this immediately. I noticed apt-get on Debian/ Ubuntu does not yet have SFML 2.6.1

Reply:
oh... building from source sounds a bit extra, I think. Can't we just run the workflows using 2.5?

Reply:
Alright. I will replace all mentions of 2.6.1 with 2.5. Including Windows download

Reply:
I think downloading the zip on Linux as well is fine. Especially for a ci test setup

Reply:
I tried the zip way for Linux but it was too much of a hustle. The CI job failed on github. you can see the last 2 recent workflow on my github Actions tab. Cmake could not locate SFML properly on Ubuntu, even with SFML_DIR given absolute path. So i just revert back to using `apt-get` which downloads SFML-2.5.1 as of today